### ISSUE TITLE
Add support for mobile

### ISSUE THREAD

**Issue by TylerGauntlett** [TylerGauntlett/universal_convention#3]
The app is not currently mobile-friendly

- [x] Allow buttons to shrink in size
- [x] Stack buttons in a logic order
- [x] Allow input/output text to stack over each other

**Comment by bharati-21** [comment]
Can you please assign this to me?

**Comment by TylerGauntlett** [comment]
@bharati-21 this issue is fairly dependent on #2

**Comment by bharati-21** [comment]
@TylerGauntlett so would you like me to work on that first?

**Comment by TylerGauntlett** [comment]
@bharati-21 Sure. Open a PR when ready

**Comment by TylerGauntlett** [comment]
@bharati-21 I added some initial mobile layout support here https://github.com/TylerGauntlett/universal_convention/pull/16. Feel free to pull the latest master and add to it though

**Comment by TylerGauntlett** [comment]
@bharati-21 Its currently non-response and looks the same on desktop and mobile. Another thing you could try is moving things around when more screen space is available. See below for ideas.

![Universal-Convention](https://user-images.githubusercontent.com/13861069/97792669-7fade500-1bb7-11eb-889b-7210b2ec3d35.png)

**Comment by TylerGauntlett** [comment]
@bharati-21 Opened new issue here for future mobile support https://github.com/TylerGauntlett/universal_convention/issues/17
